src/EditorFeatures/Core/ExternalAccess/EditorConfig/Features/Hover/HoverHandler.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.LanguageServer;
+using Microsoft.CodeAnalysis.LanguageServer.Handler;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Microsoft.VisualStudio.Text.Adornments;
+using Microsoft.CodeAnalysis.QuickInfo;
+using StreamJsonRpc;
+using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.Formatting.Rules;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.EditorConfig.Features
```

Comment:
oops I missed this in the previous PR - in a followup PR lets move this file to the `MS.CA.LanguageServer.Protocol` project into a new `EditorConfig` folder here (with namespace to match) https://github.com/dotnet/roslyn/tree/main/src/Features/LanguageServer/Protocol

Reply:
still applies - this doesn't need to be in external access